exercises/change/package.yaml

```diff
@@ -0,0 +1,20 @@
+name: change
+
+dependencies:
+  - base
+
+library:
+  exposed-modules: Change
+  source-dirs: src
+  dependencies:
+    - containers
+  # - foo       # List here the packages you
+  # - bar       # want to use in your solution.
```

Comment:
This is cosmetic, and you may or may not change, but in all the other exercises, the used packages are after the comment (those files where generated automatically by a custom script).

The only situation where this may be important, is if we write some script to automatically change those files. Other than that, it is irrelevant.